java-cfenv-boot/src/main/java/io/pivotal/cfenv/spring/boot/CfEnvironmentPostProcessor.java

```diff
@@ -78,7 +77,7 @@ public void postProcessEnvironment(ConfigurableEnvironment environment,
 
 		increaseInvocationCount();
 		if (CloudPlatform.CLOUD_FOUNDRY.isActive(environment)) {
-			CfEnv cfEnv = CfEnvSingleton.getCfEnvInstance();
+			CfEnv cfEnv = new CfEnv();
```

Comment:
Required to get CfEnv to reparse the environment. Otherwise it's cached in the singleton, which then causes failures in tests which expect CfEnv to be looking at an updated (mocked) env.